traits/trait_types.py

```diff
@@ -214,7 +221,15 @@ class Any(TraitType):
     #: A description of the type of value this trait accepts:
     info_text = "any value"
 
-    def __init__(self, default_value=NoDefaultSpecified, **metadata):
+    def __init__(
+        self,
+        default_value=NoDefaultSpecified,
+        *,
+        factory=None,
+        args=(),
+        kw={},
```

Comment:
Yes, this is considered an antipattern (compared to using `kw=None` and then explicitly checking for `None` in the body of the function). No, I don't want to change it unless people feel really strongly about it.  A default of `{}` makes the code, documentation and typing stubs simpler and clearer, and the usual concern about mutable default values doesn't apply here, because there's no plausible way that *accidental* mutation of this default value could take place.